fastlane/metadata/android/en-US/title.txt

```diff
@@ -0,0 +1 @@
+Jellyfin for Android TV
```

Comment:
We might want to rename the app to "Jellyfin for TV" to have a unified name between Fire TV and Android TV.

Reply:
I feel like I'm [bike shedding](https://en.wikipedia.org/wiki/Law_of_triviality) a bit, but I think `Jellyfin TV` is a bit easier to say out loud there's a potential renaming to make it more generic

Reply:
Jellyfin TV sounds like a streaming service and Jellyfin for TV sounds rather broad since it doesn't cover **all** TVs... so I have no idea what the right answer is here. 🤷‍♂️